distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -1791,6 +1786,35 @@ Text.parse_time_of_day self format:Date_Time_Formatter=Date_Time_Formatter.iso_t
 Text.parse_time_zone : Time_Zone ! Time_Error
 Text.parse_time_zone self = Time_Zone.parse self
 
+## ALIAS mid, slice, substring
+   GROUP Selections
+   Creates a new Text by selecting the specified range of the input.
+
+   Arguments:
+   'start' - the character position to start the substring at. 0-based index. Has support for negative values
+   'length' - the number of characters to include from start.
+
+   Returns:
+   Part of the selected Text starting at start and stopping after length.
```

Comment:
What happens if start+length goes past the end of the string?

Reply:
as long as length is >= 0 it shouldn't matter what size it is, as any length or start+length that is longer than the end of the string will simply return the full substring between start and the last character of the string.
![image](https://github.com/enso-org/enso/assets/40743109/77b57473-6a31-44a3-96ec-a39778384608)